ena-submission/scripts/get_ena_submission_list.py

```diff
@@ -213,7 +213,7 @@ def get_ena_submission_list(config_file) -> None:
     all_entries_to_submit: dict[AccessionVersion, dict[str, Any]] = {}
     for organism in config.organisms:
         ena_specific_metadata_fields = [
-            value["name"] for value in config.organisms[organism]["externalMetadata"]
+            value.name for value in config.organisms[organism].externalMetadata
```

Comment:
In general might be nice to have a more specific name for what the value is. This predates of course, just a thought reading the code